lib/cosmos/api/cmd_api.rb

```diff
@@ -1,393 +0,0 @@
-module Cosmos
```

Comment:
Where did cmd_api.rb go?

Reply:
This was an accidental checkin to the root when I was doing some git diffing against older versions